ilastik/workflows/tracking/conservation/conservationTrackingWorkflow.py

```diff
@@ -56,9 +57,10 @@ def __init__( self, shell, headless, workflow_cmdline_args, *args, **kwargs ):
         self.objectExtractionApplet = TrackingFeatureExtractionApplet(workflow=self, interactive=False,
                                                                       name="Object Feature Computation")                                                                      
         
-        self.divisionDetectionApplet = ObjectClassificationApplet(workflow=self,
-                                                                     name="Division Detection (optional)",
-                                                                     projectFileGroupName="DivisionDetection")
+        if not self.withAnimalTracking:
+            self.divisionDetectionApplet = ObjectClassificationApplet(workflow=self,
+                                                                         name="Division Detection (optional)",
+                                                                         projectFileGroupName="DivisionDetection")
```

Comment:
Sorry to nitpick, but I think the base class here should not contain the word "AnimalTracking".  Instead, I think it would be better to just define a new member function `createDvisitionDetectionApplet()`, which may return `None`.  Throughout this workflow, instead of checking for `if not self.withAnimalTracking():`, you should check for `if self.dvisionDetectionApplet:`